app/src/main/res/values/strings.xml

```diff
@@ -118,6 +123,9 @@
     <string name="theme_desc">Update KISS theme</string>
     <string name="menu_phone_create">Create contact</string>
 
+    <string name="toast_favorites_added">%s was added to favorites</string>
+    <string name="toast_favorites_already">%s is already in favorites</string>
```

Comment:
it's probably better to instead hide the menu button if the item already is in favorite, what do you think?